sql-plugin/src/main/scala/com/nvidia/spark/rapids/ColumnarOutputWriter.scala

```diff
@@ -105,12 +107,28 @@ abstract class ColumnarOutputWriter(path: String, context: TaskAttemptContext,
    */
   def write(batch: ColumnarBatch, statsTrackers: Seq[ColumnarWriteTaskStatsTracker]): Unit = {
     var needToCloseBatch = true
+    val outputTimestampType = conf.get(SQLConf.PARQUET_OUTPUT_TIMESTAMP_TYPE.key)
```

Comment:
Still looking up the config on every batch, this should be done once when the `ColumnarOutputWriter` instance is created.